### PR TITLE
Port over the latest batch score changes from the ADO repository

### PR DESCRIPTION
**Pull request by adba-msft**
As of 01/24/2024 8PM

### CODE REVIEW

**Review by Iyer-Narayan — APPROVED** [review]

File: assets/batch_score/components/driver/src/batch_score/metadata.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "component_version": "1.0.4",
+  "component_version": "1.0.3",
```
Comment (yetamsft) [inline assets/batch_score/components/driver/src/batch_score/metadata.json:2]: I assume this will be bumped to 1.1.0.

Comment (yetamsft) [reply assets/batch_score/components/driver/src/batch_score/metadata.json:2]: Please bump the version in batch_score_llm/spec.yaml too.

Comment (adba-msft) [reply assets/batch_score/components/driver/src/batch_score/metadata.json]: done, thanks

**Review by yetamsft — APPROVED** [review]
LGTM

**Pull request merged**